content/node/leanpocket.md

```diff
@@ -0,0 +1,94 @@
+---
+title: LeanPocket
+menuTitle: LeanPocket
+weight: 35
+description: Run multiple nodes on the same hardware without scaling your resource requirements.
+---
+
+LeanPocket (sometimes stylized as "LeanPOKT") is a configuration that allows multiple Pocket nodes running on a single server to consolidate their resources, resulting in better scalability and reduced hardware costs.
+
+LeanPocket is a feature available from version 0.9.2 and above.
+
+## Background
+
+Pocket Core is a single-threaded application. This means that if you wanted to run multiple Pocket nodes on the same server, each node, running its own isolated process, would each use their own resources: RAM, disk space, network throughput, etc. As many aspects of nodes use shared resources (copies of the blockchain, most notably) this was deemed very inefficient.
+
+For example, at the time of writing, a Pocket node requires a minimum of 500GB of disk space and 16GB of RAM, and that's not including the requirements of [the other chains the node may be servicing](/supported-blockchains).
+
+LeanPocket is an optimization to the core client to allow nodes to share resources, significantly reducing resource requirements. This is a non-consensus-breaking change to the client.
+
+<!-- Add charts here -->
+
+{{% notice style="info" %}}
+LeanPocket is specifically geared toward multiple nodes running on the same server. If you are running a single node, or have each of your nodes running on different server, LeanPocket will not have any benefit or effect.
```

Comment:
"If you are running a single node, or have each of your nodes running on different server"

I believe if there are folks running nodes on different servers, they can still benefit from consolidation. Why are they running multiple nodes on different servers? If it isn't for a good reason (i.e High Avaliability), then LeanPOKT would be useful for them too. 

Agreed that LeanPOKT has added benefits if only one node running.

Reply:
I just meant that if, for whatever reason, they were running one node per server and wanted to keep it that way. For sure they could benefit from consolidation. I'll make that clearer.